From 9007585c1fce21e786372c2f1ca1c7776f3eafdb Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Thu, 22 May 2008 20:54:02 +0200
Subject: [PATCH] include the proper header for XkbSetRulesDflts and XkbInitKeyboardDeviceStruct

---
 src/evdev.h |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

--- xserver-xorg-input-evdev.orig/src/evdev.h
+++ xserver-xorg-input-evdev/src/evdev.h
@@ -36,8 +36,7 @@
 #include <xf86_OSproc.h>
 
 #if defined(XKB)
-/* XXX VERY WRONG.  this is a client side header. */
-#include <X11/extensions/XKBstr.h>
+#include <xkbsrv.h>
 #endif
 
 typedef struct {
